pkg/cmd/release/create/create.go

```diff
@@ -315,6 +315,13 @@ func createRun(opts *CreateOptions) error {
 		params["draft"] = true
 	}
 
+	if gitTagAlreadyExistLocally(opts.TagName) {
+		fmt.Println("tag found locally")
+		if err := gitTagPush(opts.TagName); err != nil {
+			return fmt.Errorf("can't push tag: %v", opts.TagName)
```

Comment:
If we are unable to push we should show the `git` command error response as well so the user has more context and a possible path forward to resolving the error.